README.md

```diff
@@ -193,9 +193,25 @@ of `staged.dependencies` rather than the one loaded with `devtools::load_all()`.
 `git2r::clone` may fail. Check that the git host is reachable (VPN may be needed) and that the access token 
 has read privileges for the repositories.
 
+## Troubleshooting (For Developers of This Package)
+This package relies on `renv`. If you set environment variables such as `GITHUB_PAT` in your `~/.Rprofile`, they will
```

Comment:
When developing this package we use `renv` as users of this package don't have to use renv

Reply:
yes